modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/cluster/config/ClusterConfigUpdateReplCommand.java

```diff
@@ -39,8 +39,8 @@ public class ClusterConfigUpdateReplCommand extends BaseCommand implements Runna
     private ClusterUrlMixin clusterUrl;
 
     /** Configuration that will be updated. */
-    @Parameters(index = "0")
-    private String config;
+    @Mixin
+    private SpacedParameterMixin config;
```

Comment:
I also think that node configuration should have the same `SpacedPareameterMixin`.